checkov/secrets/plugins/ignore_secret.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from detect_secrets.util.filetype import FileType
+
+if TYPE_CHECKING:
+    from detect_secrets.util.code_snippet import CodeSnippet
+
+
+def ignore_terraform_data_block(
+    secret_line_index: int,
```

Comment:
no need for `secret_line_index` - there is `raw_context.target_index`
I wanted to use as much from code snippet as possible - if we will change how it works in the future :)

Reply:
nice catch 🚀